src/main/java/com/arcbees/analytics/client/ClientAnalytics.java

```diff
@@ -60,7 +60,7 @@ public void run() {
     };
 
     @Inject
-    ClientAnalytics(
+    public ClientAnalytics(
```

Comment:
The goal of the builder is also to reduce the risque of backward incompatibility for us. Using the CTOR directly means that if we add a parameter, it will brake someone, which isn't something we want to happen :D

using the builder, we can avoid that more easily.